doc/whats-new.rst

```diff
@@ -24,6 +24,11 @@ Enhancements
 - New documentation on :ref:`panel transition`. By
   `Maximilian Roos <https://github.com/MaximilianR>`_.
 
+- Added exclude and indexes optional parameters to align(), and exclude optional parameter
```

Comment:
Use back ticks to denote code:

   ``exclude``

and sphinx directives to like to function docstrings:

    :py:func:`~xarray.align`